database/migrations/2022_12_01_144659_create_employees_table.php

```diff
@@ -13,7 +13,7 @@
      */
     public function up()
     {
-        Schema::create('teachers', function (Blueprint $table) {
+        Schema::create('employees', function (Blueprint $table) {
```

Comment:
Vergeet de naam van het bestand ook niet te veranderen naar `employees` want dat is nu nog `create_teachers_table`

Reply:
De naam heb ik wel veranderd.